transformato/system.py

```diff
@@ -235,22 +235,22 @@ def _initialize_system(
             psf.coordinates = coord.coordinates
             # extract only ligand to generate vacuum system
 
-            g = psf.groups
-            frame_idx = []
-            frame_frame = []
-            for atom in psf.atoms:
-                if hasattr(atom, "frame"):
-                    frame_idx.append(atom.idx)
-                    frame_frame.append(atom.frame)
+            # g = psf.groups
+            # frame_idx = []
+            # frame_frame = []
+            # for atom in psf.atoms:
+            #     if hasattr(atom, "frame"):
+            #         frame_idx.append(atom.idx)
+            #         frame_frame.append(atom.frame)
 
             psf = psf[
```

Comment:
For rsfe the vacuum part is not working properly since no NUMLP section is included in the resulting psf

Reply:
@wiederm do you have an idea how to circumvent that, or should we just leave it like that and throw an error message if someone wants to do rsfe calculations with lonepairs?

Reply:
yes, I think we should through a `NonImplemented` error here.
In the long run I will try to fix this, but for now we can move forward with this I think.